pager.c

```diff
@@ -3077,7 +3082,7 @@ int mutt_pager(const char *banner, const char *fname, PagerFlags flags, struct P
         CHECK_MODE(IsEmail(extra));
         if (!assert_mailbox_writable(Contex2->mailbox)) break;
         /* L10N: CHECK_ACL */
-        CHECK_ACL(MUTT_ACL_DELETE, _("Can't delete message"));
+        if (!assert_mailbox_permissions(Contex2->mailbox, MUTT_ACL_DELETE, _("Can't delete message"))) break;
```

Comment:
When I ran `clang-format`, some of these statements ran over three lines, e.g.
```c
if (really_long_condition &&
    something_else)
  break;
```

If this happens, then the `break` needs `{}`s for clarity.

```c
if (really_long_condition &&
    something_else)
{
  break;
}
```

(or we can try to abbreviate the name of the helper function)